pyscript.core/src/config.js

```diff
@@ -46,7 +46,15 @@ const syntaxError = (type, url, { message }) => {
 };
 
 // find the shared config for all py-script elements
-let config, plugins, parsed, error, type;
+let config, type;
+
+/** @type {Promise<any> | undefined} A Promise wrapping any plugins which should be loaded. */
+let plugins;
+/** @type {any} The PyScript configuration parsed as a JSON object*. May be any of the return types of JSON.parse() ( {number | string | boolean | null | object | Array} */
```

Comment:
or TOML ;-)

Reply:
Added a comment about TOML here - thanks!

Since `JSON.parse()` and the `parse()` function from `toml-j.04` can both return an object with just about any shape, I think that `any` is still a reasonable return type, but I did adjust the comment.